src/robot_interface/models/inspection/inspection.py

```diff
@@ -17,16 +17,52 @@ class InspectionMetadata(ABC):
     start_time: datetime
     time_indexed_pose: Union[TimeIndexedPose, List[TimeIndexedPose]]
     file_type: str
-    tag_id: Optional[str] = None
-    additional: dict = None
+    tag_id: Optional[str] = field(default=None, init=False)
+    additional: Optional[dict] = field(default=None, init=False)
+
+
+@dataclass
+class AudioMetadata(InspectionMetadata):
+    time_indexed_pose: List[TimeIndexedPose]
+
+
+@dataclass
+class ImageMetadata(InspectionMetadata):
+    time_indexed_pose: TimeIndexedPose
+
+
+@dataclass
+class ThermalImageMetadata(InspectionMetadata):
+    time_indexed_pose: TimeIndexedPose
+
+
+@dataclass
+class VideoMetadata(InspectionMetadata):
+    time_indexed_pose: List[TimeIndexedPose]
 
 
 @dataclass
 class Inspection:
     id: Any
     metadata: InspectionMetadata
+    data: Optional[bytes] = field(default=None, init=False)
+
+
+@dataclass
+class Audio(Inspection):
```

Comment:
Should we have Audio and Video as Inspections when they are not supported as InspectionTasks?